### PR TITLE
Completed design of login page and pages related to it.

### LINKED ISSUE #2: Login and Register Page (Frontend)
Frontend side of Login and Registration page.

- [ ] Login page
- [ ] Registration page

### LINKED ISSUE #17: Reset your password
We need to create an UI for resetting a password. Im not sure if that should be a site or some sort of menu in the login page.

### PR DESCRIPTION
**Pull request by Myjem**
Together with @MANDRW we completed design of login, registrarion, resetting password and forgetting password page. 
Most likely, this page will be changed further in the future, I am aware that media-queries are still missing, but I think that the page is in a condition that allows us to continue working.

How to test it:

1. Start php server(php -S 0.0.0.0:8800 in terminal)
2. Open your browser and go to your localhost address with port provided in step above (127.0.0.1:8800)
3. Check login and registration page.
4. The link to the password reset page is on the login page.

I am waiting for your response and suggestions about visual side of things.

### CODE REVIEW

**Comment by xEdziu** [comment]
I am not sure about the design of buttons while hovering on them. I don't feel like "shadowing" them is a good option.
Maybe we could try doing something else - like e.g. inverting colors?

File: frontend/html/register.html [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="../styles/login.css">
+    <title>Register</title>
+    <link rel="icon" type="image/x-icon" href="../img/dark-bg-ico.ico">
+</head>
+<body>
+    <h1 class="andika-bold link"><a href="/index.php" class="link">RPG Handy Helper</a></h1>
+    <main>
+        <h2 class="andika-bold">Register your account</h2>
+        <form action="">
```
Comment (xEdziu) [inline frontend/html/register.html]: My bad, I did not provied you guys with fileds for form, so here they are:

- username
- email
- password
- confirm password
- name
- surname
- discord tag, which should be optional field in the form

File: frontend/html/register.html [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+            <label for="nickname" class="andika-bold">Nickname:</label>
+            <input type="text" placeholder="Enter Nickname" class="container" maxlength="42" name="nickname">
+            <label for="email" class="andika-bold">Email address:</label>
+            <input type="text" placeholder="Enter email address" class="container" name="email">
+            <label for="emailrepeat" class="andika-bold">Repeat your email address:</label>
+            <input type="text" placeholder="Enter email address" class="container" name="emailrepeat">
+            <label for="password" class="andika-bold">Password:</label>
+            <input type="password" placeholder="Enter passoword" class="container" name="password">
+            <label for="passowrdrepeat" class="andika-bold">Repeat your password:</label>
```
Comment (xEdziu) [inline frontend/html/register.html]: Also all of required fields in every form should have `required` attribute, so you need to change that

File: frontend/html/resetpassword.html [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="../styles/login.css">
+    <title>Reset your password</title>
+    <link rel="icon" type="image/x-icon" href="../img/dark-bg-ico.ico">
+</head>
+<body>
+    <h1 class="andika-bold link"><a href="/index.php" class="link">RPG Handy Helper</a></h1>
+    <main>
+        <form action="">
+            <label for="resetemail" class="andika-bold">Forgot your password? Give us your email address and we will send you a link to reset your password!</label>
```
Comment (xEdziu) [inline frontend/html/resetpassword.html]: this text is too bulky for me. Maybe something like:
"Forgot your password? Don't worry, we will send you a link to reset it!"

File: frontend/html/resetyourpasswordactual.html [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="../styles/login.css">
+    <title>Reset your password</title>
+    <link rel="icon" type="image/x-icon" href="../img/dark-bg-ico.ico">
+</head>
+<body>
+    <h1 class="andika-bold link"><a href="/index.php" class="link">RPG Handy Helper</a></h1>
+    <main>
+        <label for="newpassword" class="andika-bold">Your new password:</label>
+        <input type="password" name="newpassword" maxlength="42" placeholder="Enter your new passowrd" class="container">
```
Comment (xEdziu) [inline frontend/html/resetyourpasswordactual.html]: What is the purpose of `maxLength` in terms of password ?

**Review by xEdziu — CHANGES_REQUESTED** [review]

File: frontend/html/resetyourpasswordactual.html (hunk 4 above)
Comment (Myjem) [reply frontend/html/resetyourpasswordactual.html]: I didn't like how passwords could go offscreen, so I added it without giving it a second thought

**Comment by xEdziu** [comment]
Also, it would be nice if after every webpage title we could add `| RPG Handy Helper`, so it would look better than just plain "Login":
image

"Login | RPG Handy Helper" would look better in my opinion

File: frontend/html/resetyourpasswordactual.html (hunk 4 above)
Comment (xEdziu) [reply frontend/html/resetyourpasswordactual.html]: Imo, there should be another way. I am not sure, but inputs prolly natively have scrolling if the content it to long

Comment (xEdziu) [reply frontend/html/resetyourpasswordactual.html]: and you can think of a way to view the password ("the eye")

**Review by Sybirsoldat — CHANGES_REQUESTED** [review]
### Good things

Landing page looks gorgeus, login, register and resetpassword pages look ok

### To Change 

I don't like the style of buttons while hovering over them, they should be change to fit more with the style of the landing page, maybe they should change color to red just like buttons on the landing page do.

### Summary

Pages look good, but as change to style of buttons should be an easy fix i will for now deny merging

**Comment by Myjem** [comment]
I made few changes:

- Changed styling of buttons
- Added a way to view your password with an icon
- Fixed register page so it doesn't cut off any content
- Added a custom scrollbar(check register page)
- Fixed autofill background on login page
- Added asterrisks for required inputs
- Overall implemented your advices

**Review by xEdziu — APPROVED** [review]
All good now!

**Review by Sybirsoldat — APPROVED** [review]

**Pull request merged**